launcher/scripts/refracted_beam.py

```
@@ -82,36 +78,36 @@ def __init__(self, ws, material='Si', tof_bin=200, offset=0.01, pixel_size=0.000
             pars = gauss.make_params(amplitude=100, center=best_guess, sigma=5)
 
-            fit = gauss.fit(self.p_vs_t[y_pixel_min:y_pixel_max,i], pars, method='leastsq',
-                            x=self.y_pixel[y_pixel_min:y_pixel_max])
+            fit = gauss.fit(
+                self.p_vs_t[y_pixel_min:y_pixel_max, i], pars, method="leastsq", x=self.y_pixel[y_pixel_min:y_pixel_max]
+            )
 
-            a=fit.params['g_amplitude'].value
-            c=fit.params['g_center'].value
-            width=fit.params['g_sigma'].value
+            a = fit.params["g_amplitude"].value
```

Ruff (0.14.10)

Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)